autofs-5.1.0 - fix gcc5 complaints

From: Ian Kent <raven@themaw.net>

gcc5 is not happy with the way dump_core() and master_get_logopt()
are declared inline, remove the inline and let the compiler decide.
---
 CHANGELOG           |    1 +
 daemon/spawn.c      |    2 +-
 include/automount.h |    2 +-
 include/master.h    |    2 +-
 lib/master.c        |    2 +-
 5 files changed, 5 insertions(+), 4 deletions(-)

--- autofs-5.0.7.orig/CHANGELOG
+++ autofs-5.0.7/CHANGELOG
@@ -192,6 +192,7 @@
 - fix handle_mounts() termination condition check.
 - fix config old name lookup.
 - fix error handling on ldap bind fail.
+- fix gcc5 complaints.
 
 25/07/2012 autofs-5.0.7
 =======================
--- autofs-5.0.7.orig/daemon/spawn.c
+++ autofs-5.0.7/daemon/spawn.c
@@ -36,7 +36,7 @@ static pthread_mutex_t spawn_mutex = PTH
 
 #define MTAB_LOCK_RETRIES	3
 
-inline void dump_core(void)
+void dump_core(void)
 {
 	sigset_t segv;
 
--- autofs-5.0.7.orig/include/automount.h
+++ autofs-5.0.7/include/automount.h
@@ -241,7 +241,7 @@ const char **copy_argv(int argc, const c
 int compare_argv(int argc1, const char **argv1, int argc2, const char **argv2);
 int free_argv(int argc, const char **argv);
 
-inline void dump_core(void);
+void dump_core(void);
 int aquire_lock(void);
 void release_lock(void);
 int spawnl(unsigned logopt, const char *prog, ...);
--- autofs-5.0.7.orig/include/master.h
+++ autofs-5.0.7/include/master.h
@@ -120,7 +120,7 @@ void master_notify_state_change(struct m
 int master_mount_mounts(struct master *, time_t, int);
 int dump_map(struct master *, const char *, const char *);
 int master_show_mounts(struct master *);
-extern inline unsigned int master_get_logopt(void);
+unsigned int master_get_logopt(void);
 int master_list_empty(struct master *);
 int master_done(struct master *);
 int master_kill(struct master *);
--- autofs-5.0.7.orig/lib/master.c
+++ autofs-5.0.7/lib/master.c
@@ -1712,7 +1712,7 @@ int master_done(struct master *master)
 	return res;
 }
 
-inline unsigned int master_get_logopt(void)
+unsigned int master_get_logopt(void)
 {
 	return master_list ? master_list->logopt : LOGOPT_NONE;
 }
